### PR TITLE
Feature: Styling Enhancements

### PR DESCRIPTION
**Pull request by lily4178993**
# Feature: Styling Enhancements

## Description
This pull request includes multiple enhancements and improvements made to the Bookstore project.
The primary focus of this branch is on styling enhancements.

## Changes Made

1. **Styling Improvements**: Various components of the project have been styled to improve the visual appeal and user experience. These changes include updated styles for the Header component, BookForm component, BookItem component, and the Books component.
2. **Documentation**: JSDoc comments have been added to clarify code functionality, especially in the Header component.

## Reviewer Notes
To review this pull request and run the project locally, please follow these steps:

1. Clone the repository and switch to the **`bookstore-styling-milestone`** branch.
2. Create a **.env** file in the project's root directory.
3. Inside the **.env** file, add the following variables and replace them with your actual API credentials:
```
REACT_APP_URL=https://your-api-url.com/
REACT_APP_KEY=your-api-key
```
4. Save the .env file.
5. Follow the instructions provided in the **README** file to set up and run the project.

### CODE REVIEW

File: src/components/BookForm.jsx [hunk 1]
```diff
@@ -38,54 +38,58 @@ const BookForm = () => {
...
+          ))}
+        </select>
+        <Button
+          type="submit"
+          name="Add Book"
+          title="Add Book"
+          onClick={handleSubmit}
+          className="border px-4 py-1 uppercase text-xs text-white bg-secondary hover:bg-primary focus:bg-primary focus:outline-trabsparent md:hover:px-5 md:focus:px-5"
+        />
+      </form>
```
Comment (levy002) [inline src/components/BookForm.jsx]: - [ ] Kindly ensure that your `Form` is aligned correctly in order to match the provided Zeplin design.

   - ### YOUR DESIGN
  chrome-capture-2023-8-30

   - ### PROVIDED ZEPLIN DESIGN
   chrome-capture-2023-8-30 (1)

File: src/components/BooksItem.jsx [hunk 2]
```diff
@@ -17,26 +17,55 @@ const BooksItem = ({
...
+      <div className="flex items-center gap-4 md:mx-10">
+        <div className="relative w-12 h-12 rounded-full border-4 border-primary flex justify-center items-center">
+          <span className="font-roboto md:absolute left-12">
+            <span className="text-xs md:text-3xl">0%</span>
+            <br />
+            <span className="text-xs text-slate-400 hidden md:block">
+              Completed
+            </span>
+          </span>
+        </div>
```
Comment (levy002) [inline src/components/BooksItem.jsx]: - [ ] Kindly ensure leaving enough gap between the `Progressive Circle` and the progress information in order to match the provided Zeplin design.

   - ### YOUR DESIGN
 
      chrome-capture-2023-8-30 (2)

   - ### PROVIDED ZEPLIN DESIGN

  
    chrome-capture-2023-8-30 (4)

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @lily4178993   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No linter error :heavy_check_mark: 
- Well-documented Repo :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily4178993** [comment]
Thank you @levy002 for your review and feedback.

**Comment by lily4178993** [comment]
**Address review feedback**: Align Form and adjust gap between Progressive Circle and progress information.

**Review by levy002 — APPROVED** [review]
## Hello @lily4178993 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- You implemented all suggested changes :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**